src/lib/helpers/write-csv-to-s3.js

```diff
@@ -41,12 +41,12 @@ module.exports = {
 
       return fs.unlink(filePath, err => {
         if (err) {
-          logger.error(new Error(err))
+          logger.error(new Error(err.stack))
         }
         logger.info(`File no longer relevant - deleting ${dataOut.Location} from the local server`)
       })
     } catch (e) {
-      logger.error(new Error(e))
+      logger.error(new Error(e.stack))
```

Comment:
Same again. Weird that it's doing this.

```suggestion
      logger.error(e.stack)
```